scripts/demo.sh

```diff
@@ -14,6 +15,34 @@ kubeconfigs_dir="${work_dir}/kubeconfigs"
 echo "Switch to hub cluster ${hub}"
 export KUBECONFIG="${kubeconfigs_dir}/kind-config-${hub}/kubeconfig"
 
+function patch_submariner_operator() {
+    local cluster=$1
+    echo "Waiting for submariner manifestworks to be ready for ${cluster}"
+    kubectl wait --for=condition=available manifestworks submariner-operator -n ${cluster} --timeout=60s
+    kubectl wait --for=condition=available manifestworks submariner-resource -n ${cluster} --timeout=60s
+
+    echo "Waiting to patch submariner-operator deployment on ${cluster}"
+    kubectl annotate submarinerconfig submariner -n ${cluster} skipOperatorGroup=""
+
+    submrepo="quay.io/submariner"
+    submver=0.14.0-rc2
```

Comment:
Why not 0.14.0?

Reply:
Ha, forgot to update. Was testing scripts before 0.14.0 as available :)

Reply:
Also this is redundant as we are declaring and sourcing them from scripts/vars.sh